Src/BootCamp.Chapter/Logging/Logger.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace BootCamp.Chapter.Logging
+{
+    public class Logger : ILogger
```

Comment:
This class is not needed. You have different types of logger, container logger, in this case, serves no purpose. It could work, but no reason to use it like that.

Reply:
I'm actually really glad you threw this back at me 😄 It made me realize that instead of using the Logger container and the target types, I could have just used ILogger in all my functions instead of passing in a specific logger type.

It was so obvious but I didn't see that. Thanks sensei!